packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -49,6 +49,36 @@ export interface ClusterEngineConfig {
    * @default - use the default port for clusters (3306)
    */
   readonly port?: number;
+
+  /**
+   * Features supported by the database engine.
+   *
+   * @see https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DBEngineVersion.html
+   *
+   * @default - no features
+   */
+  readonly features?: EngineFeatures;
+}
+
+/**
+ * Represents Database Engine features
+ */
+export interface EngineFeatures {
+  /**
+   * Feature for the DB instance that the IAM role to access the S3 bucket for import
```

Comment:
I think it's "Name of the feature (...)", not just "Feature (...)".

Reply:
good call